order/README.md

```diff
@@ -6,28 +6,35 @@ Ordering with loosely defined order values will give the provider more freedom t
 
 # POST /order
 
-## Request
+## Create Order Request
 | Field Name | Type                                                                       | Description |
 | ---------- | -------------------------------------------------------------------------- | ----------- |
 | datetime       | string                                                                     | **REQUIRED.** Datetime field is a [ISO8601 Time Interval](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) |
 | product_id         | string                                                                     | **REQUIRED.** Product identifier. The ID should be unique and is a reference to the constraints which can be used in the constraints field. |
 | geometry   | [GeoJSON Geometry Object](https://tools.ietf.org/html/rfc7946#section-3.1) | **REQUIRED.** Defines the full footprint of the asset represented by this item, formatted according to [RFC 7946, section 3.1](https://tools.ietf.org/html/rfc7946#section-3.1). The footprint should be the default GeoJSON geometry, though additional geometries can be included. Coordinates are specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). |
 | filter | CQL2 JSON | A set of additional constraints in [CQL2 JSON](https://docs.ogc.org/DRAFTS/21-065.html) based on the constraints exposed in the product. |
 
-## Response
+## Create Order Response
+See [Order Object](#order-object).
+
+# GET /orders
```

Comment:
You can't have multiple top-level headers.

Reply:
that's a markdown rule or a STAC/T rule?

Reply:
I think that's a Markdown rule, at least my linter complains ;)

Reply:
Do you have more linter rules disabled locally than are in `.markdownlint.json`? Using the linter there seem to be a number of remaining problems.

Reply:
No, my linter doesn't seem to use this config file at all.